add Html form

bhushan354 opened the pull request:
- added html form properly.
- added connect icons .

Thank you ,
Bhushan

bhushan354 committed:
changes done

bhushan354 committed:
changes

bhushan354 committed:
fix Horizontal scrolling

bhushan354 committed:
update readme file in accordance with changes suggested by previous reviewer

bhushan354 committed:
fix Horizontal scrolling

bhushan354 committed:
Update style.css

maserekafelex committed:
Add icon in style.css

bhushan354 committed:
fix linters error

bhushan354 committed:
Merge branch 'main' into htmlForm

V-Blaze commented on index.html:
```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            class="firstForm"
+            id="form"
+            method="post"
+            action="https://formspree.io/f/xgedqoye"
+          >
+
+          
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
```
- You have done great so far. :+1: :+1: But there are some changes you still need to make before moving to the next milestone.
- [x] As part of the requirement, you are meant to replicate the Figma design as much as possible. Looking at your UI, I think you need to adjust the height of the inputs to match the Figm design. also, the padding by the side of the form is much compared to the one in the Figma design. I'll suggest you adjust this. :smile: :wave: 

- - Your design:
![Selection_082](https://github.com/bhushan354/My-Portfolio/assets/107632780/259e94a9-bbc9-4a85-9eb1-2ba72cf330e9)

- - Figma design:
![Screenshot from 2023-07-12 15-58-36](https://github.com/bhushan354/My-Portfolio/assets/107632780/a53c8619-8ab4-4d93-bb40-1579878ea5bf)

- [x] Following the standard practices, you are not meant to capitalize the first letter of the `type and name ` attribute in your form input. the correct attributes should be written as `type="text" name="name"`  I'll suggest you update this in your inputs. :+1: :+1:

V-Blaze commented on index.html:
```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+                <div class="gaps1"></div>
+              </li>
+
+              <li class="newInputName01">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="First name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+              <li class="newInputName02">
+                <input
+                  name="LastName"
+                  type="Text"
+                  id="Name"
+                  placeholder="Last name"
+                  maxlength="30"
+                  required
+                />
+                <div class="gaps11"></div>
+              </li>
```
- [x] I'll suggest you remove the two extra inputs `first name and last name` as they are not visible in the Ui and will prevent the form from actually submitting. :+1: :+1:

V-Blaze commented on index.html:
```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+                <div class="gaps11"></div>
+              </li>
+
+              <li class="newInputEmail">
+                <input
+                  name="Email"
+                  type="Email"
+                  id="Email"
+                  placeholder="Email address"
+                  maxlength="30"
+                  required
+                />
```
- [x] You added a `maxlength` attribute to the email input, this is not part of the requirement, your email is not meant to have a `maxlength`, I'll suggest you remove this. :+1: :100:

V-Blaze commented on index.html:
```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+              </li>
+
+              <li>
+                <button class="submit" type="submit" value="submit">
```
> Implement the button interactions (enable, hover, etc.).

- [ ] The button interactions like the hover is not implemented in this submit button as stated in the project requirement. I'll suggest you look at the Figma designs dn get the text and background color for different button states. :wave: 

- - FIGMA:
![Screenshot from 2023-07-11 13-50-12](https://github.com/chiedozie013/portfolio-microverse/assets/107632780/d01f64d5-f311-452d-ba84-6d51ed13585f)
- [ ] When I try to submit a form, I get this error in the console. I'll suggest you look into this and fix it. And also try and make sure the form submits from your end to `form-spree` before request ing the next review. :smile: 

- - Console error screenshot:
![Screenshot from 2023-07-12 16-01-40](https://github.com/bhushan354/My-Portfolio/assets/107632780/335fca03-2b06-488b-b102-b7d07c163fa2)

  
- [ ] Also I'll suggest you adjust the width of this button to fully match the Figma design as your button width is a bit smaller than the given design: :+1: :+1:

V-Blaze reviewed (changes requested):
## Hi @bhushan354  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

![giphy](https://user-images.githubusercontent.com/107632780/229314554-3700de13-2c00-46d5-bc77-46ad09af7467.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Github flow✔
-  Linters setup correctly👋 
-  Readme looks professional✔

 ## Required Changes ♻️

## Optional suggestions
- [OPTIONAL] TO MAKE YOUR FILE AND PROJECT MORE ORGANIZED, I'LL SUGGEST YOU CREATE A NEW FOLDER WITH THE NAME `images` AND THEN MOVE ALL THE IMAGES INOT THE FOLDER. IT WILL MAKE YOUR FILE STRUCTURE LOOK BETTER. ALSO, REMEMBER TO UPDATE THE `URL` OF THE IMAGES IN YOUR CODE. :+1: :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bhushan354 committed:
implement changes

bhushan354 committed:
Merge branch 'htmlForm' of github.com:bhushan354/My-Portfolio into htmlForm

bhushan354 committed:
linters fix

bhushan354 commented:
Hello sir , Thanks for review my PR I have made changes according to your review.

- Adjusted the height and width of the inputs to match the Figma design.
- Added hover to button and changed width and height of button according to figma design.
- Removed unnecessary code that was preventing form from submitting.
- Removed maxlength attribute.
- The form now gets submitted succesfully to form-spree.
- Fixed the error that were showing in console while submitting the form.

Thank you ,
Bhushan 😄

bhushan354 committed:
linters fix

shahadat3669 commented on index.html:
```diff
@@ -93,3 +93,424 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="name"
+                  type="text"
+                  id="Name"
+                  placeholder="Full name"
+                  required
+                />
```
- [ ] Great job on implementing the input fields! However, as mentioned in the previous review, the height of the input fields needs to be adjusted to match the design. Please refer to the design specifications and update your code accordingly. Using padding can be a suitable approach to achieve the desired height. Keep up the good work! 💪🎉
![image](https://github.com/bhushan354/My-Portfolio/assets/55840999/39f48245-48db-4f10-9c2a-a9a6538a0eb3)

shahadat3669 commented on index.html:
```diff
@@ -93,3 +93,424 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+              <li class="newInputName">
+                <input
+                  name="name"
+                  type="text"
+                  id="Name"
+                  placeholder="Full name"
+                  required
+                />
+                <div class="gaps1"></div>
+              </li>
+
+              <li class="newInputEmail">
+                <input
+                  name="Email"
+                  type="Email"
+                  id="Email"
+                  placeholder="Email address"
+                  required
+                />
+                <div class="gaps2"></div>
+              </li>
+
+              <li class="textArea" id="message">
+                <textarea
+                  class="textArea1"
+                  name="message"
+                  id="textArea1"
+                  placeholder="Write me something..."
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+
+              <li>
+                <button class="submit" type="submit" value="submit">
+                  Get In Touch
+                </button>
+              </li>
+            </div>
+            </ul>
```
- [ ] Great job on incorporating flexbox for the input elements! However, as mentioned in the previous review, the gap between the input elements needs to be adjusted to match the design specifications. Currently, the margin is set to `16px`, but it should be `24px`. To achieve the desired layout, I recommend using flexbox along with the appropriate gap properties or margins to create the desired spacing between the input elements. Keep up the great work! 💪🌟

![image](https://github.com/bhushan354/My-Portfolio/assets/55840999/426afce0-d054-4441-a821-a84f35a865ed)

shahadat3669 commented on index.html:
```diff
@@ -93,3 +93,424 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+              </li>
+
+              <li class="newInputEmail">
+                <input
+                  name="Email"
+                  type="Email"
+                  id="Email"
+                  placeholder="Email address"
+                  required
+                />
```
- [x] Great job on updating the "name" field based on the previous reviewer's request! However, please make sure to apply the same changes to the "email" field as well. The correct attributes for the email field should be `type="email"` and `name="email"`. This will ensure proper form validation and data handling. Keep up the good work! 👏🌟

shahadat3669 commented on index.html:
```diff
@@ -93,3 +93,424 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+      <div class="form">
+        <a href="#form"></a>
+        <div class="mainContent">
+          <h2 class="smallPara">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
```
- [ ] [Optional] Good job on adding the title element! However, please adjust the spacing for the title to match the design specifications. As per the design, there should be a 24px gap on the left and right sides of the title. This will ensure consistency with the overall layout and spacing. Make the necessary adjustments to your code accordingly. Keep up the great work! 👍🌟

shahadat3669 reviewed (changes requested):
Hi @bhushan354, @maserekafelex , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
<img src="https://media2.giphy.com/media/m3864rBwwBTKMAbpn3/giphy.gif"/>

 ## Highlights
✅ No linter errors
✅ Good PR title
✅ Good PR Description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

bhushan354 committed:
changes

bhushan354 commented:
Hello sir , thank you for reviewing my project , I have made necessary changes suggested by previous reviewer and I hope this will meet all requirement . 

I just wanted to let you know that this is my last review left so if there is any small change required , it would be of great help to me if you message me in slack about that .

Thank you for consideration ,
Bhushan

Shedrack-Sunday reviewed (approved):
Hi @bhushan354    ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

![image](https://github.com/krisbobo/blog-app/assets/50990925/76d09bb1-9dc7-46c0-be49-31f366afa4b8)


Highlights
- [x] Good code organisation  ✔️
- [x] Nice use CSS styles ✔️ 
- [x] Kudus on heeding to the last reviewer's comments ✔️


Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

bhushan354 commented:
@Shedrack-Sunday Thank you sir , means a lot 😃

bhushan354 committed:
changes to desktop version

bhushan354 committed:
linters
